components/inputs/docs/input-number.md

```diff
@@ -0,0 +1,52 @@
+# Number Inputs
+
+The `<d2l-input-number>` element is a wrapper around the `<d2l-input-text>` component. It's intended for inputting numbers only.
```

Comment:
We may choose one day to not simply wrap `d2l-input-text`, so I don't think we need to leak this implementation details. You could say something like:
> The `<d2l-input-number>` element is similar to `<d2l-input-text>`, except it's intended for inputting numbers only.